### DISCUSSION TITLE
langium boolean properties confused

### DISCUSSION THREAD

**Discussion by zrwsmd** [eclipse-langium/langium#1280] [Q&A] [answered]
for this part(https://langium.org/docs/grammar-language/),i am a little confused:
         

![image](https://github.com/eclipse-langium/langium/assets/26995040/e9350914-9cfa-4847-9887-cfad63d5cde0)
`

if i want to set remote to true,i might write like employee smith remote,but if i want to  set remote to false,how should i write it,it seems that remote keyword is necessary

**Comment by msujew** [comment]
@zrwsmd Langium's boolean properties are automatically assigned to `false` if they're not true. No need for any action.

**Reply by zrwsmd** [reply]
so for below example,how do i write it,it seems that if i write 'employee smith',there is a parse error
write ‘'employee smith remote' indicate set remote property  to true,am i right?

**Reply by msujew** [reply, marked as answer]
Is the `?` at the end of the assignment missing? As the documentation explains, the `?` cardinality makes the `remote` part optional.

**Reply by zrwsmd** [reply]
oh,gosh,i remember i add the last ?, but .langium file seem did  not add it,very thanks
